.gitignore

```diff
@@ -52,7 +52,7 @@ BenchmarkDotNet.Artifacts/
 project.lock.json
 project.fragment.lock.json
 artifacts/
-**/Properties/launchSettings.json
+# **/Properties/launchSettings.json
```

Comment:
Was the gitignore causing issues? I'm curious about the lines in comments

Reply:
The problem is that if you used the command git clean -fdx, it would delete that file and it wouldnt regenerate it, making you unable to build anymore on WinUI. The only way I found to fix that was to copy the file from another project, but I, and Thomas just felt that commenting that line would make things easier.

Reply:
Since it's necessary now we can remove the line from gitignore?
@jeanplevesque

Reply:
yeah we should remove it from the gitignore if the file is necessary.